### PR TITLE
Support platforms which have single-RE ISSU.

### PR DESCRIPTION
**Pull request by stacywsmith**
Support platforms which have single-RE ISSU.
Handle the fact that some platforms don't return <package-result> from sw install.
Unit test fixup and PEP8 cleanup.

### CODE REVIEW

**Comment by stacywsmith** [comment]
ok to test

**Comment by jnpr-community-netdev** [comment]
Can one of the admins verify this patch?

File: lib/jnpr/junos/utils/sw.py [hunk 1]
```diff
@@ -241,7 +243,9 @@ def pkgaddISSU(self, remote_package, **kvargs):
 
     def _parse_pkgadd_response(self, rsp):
         got = rsp.getparent()
-        rc = int(got.findtext('package-result').strip())
+        # If <package-result> is not present, then assume success.
+        # That is, assume <package-result>0</package-result>
```
Comment (vnitinv) [inline lib/jnpr/junos/utils/sw.py]: Is this valid assumption?

File: lib/jnpr/junos/utils/sw.py [hunk 2]
```diff
@@ -70,6 +70,8 @@ def __init__(self, dev):
             self._multi_RE is True and dev.facts.get('vc_capable') is True and
             dev.facts.get('vc_mode') != 'Disabled')
         self._mixed_VC = bool(dev.facts.get('vc_mode') == 'Mixed')
+        self._single_re_issu = bool('current_re' in dev.facts and
+                                    'localre' in dev.facts['current_re'])
```
Comment (vnitinv) [inline lib/jnpr/junos/utils/sw.py]: Can I get any doc reference for "localre"?

File: lib/jnpr/junos/utils/sw.py (hunk 1 above)
Comment (stacywsmith) [inline lib/jnpr/junos/utils/sw.py]: I can't guarantee it's a perfect assumption, but I do believe it's valid. In my experience, platforms which do not include `<package-result>` will include an `<rpc-error>` element if there's an error. We'll raise an `RpcError` exception in that case.

File: lib/jnpr/junos/utils/sw.py (hunk 2 above)
Comment (stacywsmith) [inline lib/jnpr/junos/utils/sw.py]:
```
{master:0}
user@s0> file show /etc/hosts.junos | match localre 
128.0.0.63       localre
```
The platforms which support single-RE ISSU use this internal IP address for communication during the ISSU process.

**Comment by stacywsmith** [comment]
I'm fine waiting until after 2.1.4 release to merge this.

File: lib/jnpr/junos/utils/sw.py (hunk 1 above)
Comment (vnitinv) [inline lib/jnpr/junos/utils/sw.py]: @stacywsmith Please add a log for users reference when <package-result> is missing

Comment (stacywsmith) [inline lib/jnpr/junos/utils/sw.py]: @vnitinv Log added.

**Pull request merged**